include/xten/Dialect/XTenNN/IR/XTenNNOps.td

```diff
@@ -251,6 +251,42 @@ def XtenNN_MishOp: XTenNN_Op<"mish", [Pure, TosaExtension, ElementwiseUnary, Sam
   let assemblyFormat = [{ operands attr-dict `:` functional-type(operands, results) }];
 }
 
+def XtenNN_ResizeOp: XTenNN_Op<"resize", [Pure, TosaExtension]> {
+  let summary = "Performs an upscale of 2 the the image.";
```

Comment:
I don't think it would be up to the operator to decide that the upscale is 2, this is a constraint from the kernel but it should be up to the pattern matching to decide that the op can't be matched to the kernel (And if it changes in the future, that would make it a hassle if we need to modify the op definition along with its lowerings)

Reply:
Done